src/java/org/apache/cassandra/io/sstable/format/SSTableReader.java

```diff
@@ -530,8 +529,7 @@ public static SSTableReader open(Descriptor descriptor,
         }
 
         long fileLength = new File(descriptor.filenameFor(Component.DATA)).length();
-        if (logger.isDebugEnabled())
-            logger.debug("Opening {} ({})", descriptor, FBUtilities.prettyPrintMemory(fileLength));
+        logger.info("Opening {} ({})", descriptor, FBUtilities.prettyPrintMemory(fileLength));
```

Comment:
The startup log could get clogged really quickly if there are tables with a lot of files (like those using LCS, which can end up with thousands of sstables), making it hard to read.
What would you need that information for at INFO level?

Reply:
Yes, you could have a few thousand lines of logs, but in the grand scheme of things that's pretty minor compared to the benefit of knowing if the DB started up correctly.  After startup you wouldn't see this very often.  This goes back to the idea of putting INFO logging in to be able to understand correctness after the fact.  If the DB starts up and you don't have this information readily available it's hard to answer questions like "did Cassandra see SSTable XYZ?"  I can think of a few cases in the past when helping teams with questions of data loss that this has come up. 

I've run a server with this logging enabled, and after startup it's rarely seen.  I think it's an important part of the startup checks.  I'd definitely want it on any servers that I would later try to diagnose with a potential data loss issue.

Reply:
I ran a test where I ran tlp-stress against my node locally with compaction disabled, and created 3K sstables, then started things back up.

Here's a search of the system log for the opening lines:
```
rg Opening system.log | wc -l                                                                                                                                      
    3030
```

I saved the log messages themselves to a separate file:
```
rg Opening system.log > opening.log
```

The total size of the file is 615KB.  Given the importance of being able to diagnose data loss problems after the fact, I'd say this is a pretty small price to pay for the ability to go back and figure out if the server started up correctly.  What do you think?

Reply:
I can't think of a time where this would've been useful to diagnose an issue, which doesn't mean such a case doesn't exist. 
My 2 cents on this is that so many lines will bother me as startup logging is already a lot, but I know what to search if I want to move quickly to a specific line, so I can deal with it.
If you have cases in mind where this would be helpful, fair enough.

Reply:
I ran into a few cases at TLP were folks were concerned data was missing.  My diagnosis after the fact was difficult because I didn't know what SSTables had been opened.